backend/package.json

```diff
@@ -15,9 +15,12 @@
     "dotenv": "^16.4.5",
     "express": "^4.19.2",
     "mongodb": "^6.5.0",
-    "nodemon": "^3.1.0"
+    "nodemon": "^3.1.0",
+    "ts-node": "^10.9.2"
```

Comment:
`nodemon` (https://stackoverflow.com/a/69811128) and `ts-node` (https://typestrong.org/ts-node/docs/installation) should be `devDependencies`.

You can learn more about the difference between `dependencies` and `devDependencies` [here](https://stackoverflow.com/a/49035993)

Reply:
I just updated the code!